### PR TITLE
Some improvements to blue-app-monero.rst

### PR DESCRIPTION
**Pull request by jonathancross**
* Renaming folder "developper" to "developer".

Changes in blue-app-monero.rst
* Normalizing spelling of "pre-hash" => "prehash".
* Various typo and spelling fixes.

### CODE REVIEW

File: doc/developer/blue-app-monero.rst [hunk 1]
```diff
@@ -408,7 +407,7 @@ to indicate another command will be sent.
 +---------------+----------------------------------------------------------+
 | ``x--------`` | Last sub command indicator                               |
 |               |                                                          |
-| ``1--------`` | More identical subcommand to comme                       |
```
Comment (jonathancross) [inline doc/developer/blue-app-monero.rst]: Not sure what was meant by "comme" here, but I assumed it was supposed to be "command".  Please let me know if this is incorrect.

Comment (dEBRUYNE-1) [reply doc/developer/blue-app-monero.rst]: I think it's meant as "to come", so "More identical subcommand to come". Perhaps "More identical subcommand forthcoming" would be better.

**Comment by cslashm** [comment]
Thanks for the review
Yes it was "come". I fixed it with dEBRUYNE-1 suggestion.

**Comment by jonathancross** [comment]
Cheers  :-)

**Pull request merged**